### PR TITLE
Ataymano/python safe constructor

### PR DESCRIPTION
**Pull request by ataymano**
do not throw on destructor if constructor was not successful

### CODE REVIEW

**Comment by ataymano** [comment]
#2820

File: python/vowpalwabbit/pyvw.py [hunk 1]
```diff
@@ -277,6 +277,10 @@ class vw(pylibvw.vw):
     """The pyvw.vw object is a (trivial) wrapper around the pylibvw.vw
     object; you're probably best off using this directly and ignoring
     the pylibvw.vw structure entirely."""
+    log_wrapper = None
```
Comment (jackgerrits) [inline python/vowpalwabbit/pyvw.py:280]: Does this work outside of the `__init__` function?

Comment (ataymano) [reply python/vowpalwabbit/pyvw.py:280]: yes, it is called class attributes

Comment (ataymano) [reply python/vowpalwabbit/pyvw.py:280]: actually I see that I haven't moved all of them here (at least forgot log_fwd). If approach is good, will send update and make it more consistent

Comment (jackgerrits) [reply python/vowpalwabbit/pyvw.py:280]: Which version were that added in?

Comment (ataymano) [reply python/vowpalwabbit/pyvw.py:280]: I do not see version where it is unavailable. 2.7, 3.5+ all have it.

Comment (jackgerrits) [reply python/vowpalwabbit/pyvw.py:280]: Awesome!

**Review by jackgerrits — APPROVED** [review]

**Pull request merged**